src/handlers/comment/handlers/index.ts

```diff
@@ -37,6 +45,21 @@ export * from "./help";
 export * from "./multiplier";
 export * from "./query";
 
+export interface RewardsResponse {
+  error: string | null;
```

Comment:
can you also replace this error with `throw Error`?

Reply:
> can you also replace this error with `throw Error`?

It's thrown here https://github.com/ubiquity/ubiquibot/blob/1d9cfdcc754cc858c450de9bd759555bde8884b8/src/handlers/comment/handlers/index.ts#L105

 Looks better to return errors and throw it on on call as its neater and doesn't cause the assertion like before